src/auth/oauth/test/oauth.test.ts

```diff
@@ -370,21 +370,11 @@ describe('validateAuthCallback', () => {
     testCallbackQuery.hmac = expectedHmac;
 
     fetchMock.mockResponse(JSON.stringify(successResponse));
-    await ShopifyOAuth.validateAuthCallback(req, res, testCallbackQuery);
+    const returnedCookie = await ShopifyOAuth.validateAuthCallback(req, res, testCallbackQuery);
```

Comment:
This is great, but I feel like `returnedCookie` doesn't really represent this.
```suggestion
    const currentSession = await ShopifyOAuth.validateAuthCallback(req, res, testCallbackQuery);
```